### PR TITLE
Change Copyright Notice to Be Shared Among Many Contributors.

### PR DESCRIPTION
**Pull request by blueandgold**
Since we have external contributions, "Copyright Google" becomes obsolete. So, this means the copyright notice needs to be updated so that the copyright of the collective work is shared among the contributors.

This change is based on these examples:
https://github.com/chromium/crashpad/blob/master/AUTHORS
https://github.com/chromium/crashpad/blob/master/CONTRIBUTORS
https://github.com/chromium/crashpad/blob/master/crashpad.gyp

### CODE REVIEW

File: AUTHORS [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# This is the official list of Forseti Security authors for copyright purposes.
+# This file is distinct from the CONTRIBUTORS files.
+# See the latter for an explanation.
+
+# Names should be added to this file as:
+# Name or Organization <email address>
+# The email address is not required for organizations.
+
+Google Inc.
+Spotify AB
```
Comment (carise) [inline AUTHORS]: Do we add Denzel here as well?

Comment (blueandgold) [reply AUTHORS]: We can.  But according to the link below, it seems like he would have to initiate the request to be a copyright holder.
https://g3doc.corp.google.com/company/teams/opensource/releasing/authors.md?cl=head

Comment (carise) [reply AUTHORS]: Makes sense. Removed the internal link, btw

File: CONTRIBUTORS [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+#
+# Names should be added to this file as:
+#     Name <email address or github username>
+
+Adam Cotenoff <acotenoff>
+Andrew Hoying <ahoying>
+Anton Sapozhnikov <snowytoxa>
+Carise Fernandez <carise>
+Carly Schneider <cschnei3)
+Denzel Morris
```
Comment (mirons-google) [inline CONTRIBUTORS]: denzel-morris

**Pull request merged**